index.html

```diff
@@ -32,7 +32,8 @@
 
 <body>
   <div class="container">
-    The current universal constant is: DRAGONFRUIT.
+    <p>The current universal constant is: DRAGONFRUIT.</p>
+    <p>The current universal currency is: POTATO.</p>
```

Comment:
I have concerns about the impact this will have on the economy, but otherwise LGTM